packages/tools/test/cpu_RectangleROI_test.js

```diff
@@ -370,8 +365,8 @@ describe('RectangleROITool (CPU):', () => {
     const addEventListenerForAnnotationRendered = () => {
       element.addEventListener(csToolsEvents.ANNOTATION_RENDERED, () => {
         const rectangleAnnotations = annotation.state.getAnnotations(
-          element,
-          RectangleROITool.toolName
+          RectangleROITool.toolName,
+          getEnabledElement(element).FrameOfReferenceUID
```

Comment:
I would rather this have been a fixed function, eg:
getAnnotationManagerId: (element):string => getEnabledElement(element).FrameOfReferenceUID
as that provides a single place to change everything once in order to add more tyeps of annotations managers.